src/components/Breadcrumb/BreadcrumbItem.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { BreadcrumbItem as CarbonBreadcrumbItem } from 'carbon-components-react';
+
+const BreadcrumbItem = () => {
+  return <CarbonBreadcrumbItem />;
+};
+
+export default BreadcrumbItem;
```

Comment:
I don't think there's a need to redefine and export the component with no changes. 

I think it can be a one liner in this file
```js
export default Breadcrumb from `carbon-components-react`;
// or it might need to be
export default from 'carbon-components-react/lib/components/BreadcrumbItem';
```

I'm not exactly sure on the syntax.